.github/workflows/appsignals-e2e-test.yml

```diff
@@ -156,55 +165,63 @@ jobs:
             sleep 10
           done
 
+      # This steps increases the speed of the validation by creating the telemetry data in advance
+      - name: Call all test APIs
+        continue-on-error: true
+        run: |
+          curl -S -s -o /dev/null http://${{ env.APP_ENDPOINT }}/outgoing-http-call/
+          curl -S -s -o /dev/null http://${{ env.APP_ENDPOINT }}/aws-sdk-call/
+          curl -S -s -o /dev/null http://${{ env.APP_ENDPOINT }}/remote-service?ip=${{ env.REMOTE_SERVICE_POD_IP }}/
+          curl -S -s -o /dev/null http://${{ env.APP_ENDPOINT }}/client-call/
```

Comment:
Where were these being invoked before this change? I don't see any old steps doing this.

Reply:
They are being invoked in the validator (and still are), but this just helps speed it up because while gradle is building the package the telemetry has time to appear in CloudWatch.

Reply:
Makes sense.